lxd/devlxd_storage.go

```diff
@@ -337,7 +337,8 @@ func devLXDStoragePoolVolumePutHandler(d *Daemon, r *http.Request) response.Resp
 	}
 
 	// Ensure the volume owner cannot be changed.
-	if !isDevLXDVolumeOwner(vol.Config, identity.Identifier) {
+	owner, ok := vol.Config["volatile.devlxd.owner"]
```

Comment:
why not using helper function here?

Reply:
I prefer the approach you proposed that uses isDevLXDVolumeOwner() so we dont duplicate logic.

Reply:
And we can see all use-case of checking device ownership.